tools/wake-format/emitter.cpp

```diff
@@ -580,7 +580,7 @@ wcl::doc Emitter::walk(ctx_t ctx, CSTElement node) {
       is_floating_comment,
       fmt()
           .fmt_while(TOKEN_COMMENT,
-                     fmt().token(TOKEN_COMMENT).freshline().fmt_if(TOKEN_NL, fmt().next()))
+                     fmt().walk(WALK_TOKEN).freshline().fmt_if(TOKEN_NL, fmt().next()))
```

Comment:
Why do we need this to be duplicated? Can't we just use the TokenReplaceAction?

Reply:
ah it shouldn't be. I only thought it was needed because I missed some cases in TokenReplaceAction